ack_vuln.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/python3
+# Cisco Cyber Vision V4.x and V5.0.x
+# Vulnerability Management
+# Version 1.3 - 2023-01-09
+import argparse
+import csv
+from json import dumps
+
+import cvconfig
+import api
+
+
+# TODO: Throw error if cve-id is not provided
+def main():
+    parser = argparse.ArgumentParser(
+        prog="ack_vuln.py",
+        description="Vulnerabilities Management")
+
+    # Environment Options parsing
+    parser.add_argument("--token", dest="token",
+                        help="Use this token")
+    parser.add_argument("--center-ip", dest="center_ip",
+                        help="Specified the center FQDN or IPv4 address (default:'cybervision')")
+    parser.add_argument("--center-port", dest="center_port", default=cvconfig.center_port,
+                        help="Specified the center port (default: %d)" % cvconfig.center_port)
+
+    # Main Command Parsing
+    command_group = parser.add_mutually_exclusive_group()
+    command_group.add_argument("--export-affected-devices", dest="command_export", default=False,
+                               action="store_true",
+                               help="Export all affected devices of a given preset into a CSV file, If Preset name is not passed then \"All data\" preset is considered as default\n")
+    command_group.add_argument("--ack-by-preset", dest="command_ack_by_preset", default=False,
+                               action="store_true",
+                               help="Update all devices in this preset for given CVE ID, If Preset name is not passed then \"All data\" preset is considered as default\n")
+    command_group.add_argument("--ack-by-devices", dest="command_ack_by_devices", default=False,
+                               action="store_true",
+                               help="Update all devices from a CSV file for given CVE ID\n")
```

Comment:
Should this say "Update all devices" (wherein we don't say what we are updating) or "Acknowledge the CVE ID for all devices from a CSV file" ?